packages/components/src/Select/docs/customValue.tsx

```diff
@@ -0,0 +1,39 @@
+import { Select, Label, Avatar, Text } from "@hopper-ui/components";
+
+import { users, type User } from "./data.ts";
+
+export default function Example() {
+    return (
+        <Select 
+            fieldChildren={<Label>Users</Label>}
+            items={users}
+            renderValue={({ defaultChildren, selectedItem }) => {
+                if (selectedItem) {
+                    const user = selectedItem as User;
+
+                    return (
+                        <>
+                            <Avatar key={`avatar_${user.id}`} name={user.name} src={user.avatar} />
+                            <Text slot="label">{user.name}</Text>
+                        </>
+                    );
+                }
+
+                return defaultChildren;
+            }}
+            defaultSelectedKey={users[0].id}
+        >
+            {item => {
```

Comment:
`{(item: User) => ...}` instead of casting won't work?

I would also suggest to deconstruct user to avoid repeating the word.

Reply:
it does work, thanks.